src/gam/gapi/cloudidentity/groups.py

```diff
@@ -171,6 +174,10 @@ def print_():
         if myarg == 'todrive':
             todrive = True
             i += 1
+        elif myarg == 'member':
```

Comment:
Did you get this working? It's just throwing a 400 error for me.

It's going to be very confusing to non-Enterprise admins when a command like:
```
gam print cigroups
```
works but
```
gam print cigroups member someuser@domain.com
```
doesn't. We need to:
1. make these entirely separate commands, not arguments to an existing command (that will simplify the if/then/else in each function also)
1. try to catch the error Google is throwing for non-Enterprise accounts and inform the admin that the issue is that they don't have Enterprise licenses. That should head off a LOT of discussion group questions.

same thing for the membership commands below. I don't have immediate thoughts on what the new GAM commands should look like so open to suggestions.